xtask/codegen/css.ungram

```diff
@@ -1210,26 +1212,10 @@ AnyCssUrlValue =
 	CssUrlValueRaw
 	| CssString
 
-
-CssVarFunction =
-	'var'
-	'('
-		property: CssCustomProperty
-		value: CssVarFunctionValue?
-	')'
-
-CssVarFunctionValue =
-	','
-	value: CssIdentifier
```

Comment:
I removed these nodes because they aren't used anymore now that functions in general are more broadly supported.